packages/ui/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["index.ts", "src", "types"],
+  "include": ["index.ts", "src", "types", "scssDeclaration.d.ts"],
```

Comment:
This is already included in root tsconfig, so it should either be just here or just there.